src/design-tokens/tier-1-definitions/typography.json

```diff
@@ -0,0 +1,95 @@
+{
+  "eds": {
+    "font-family": {
+      "primary": {
+        "value": "'Arimo', sans-serif"
+      },
+      "secondary": {
+        "value": "'Open Sans', sans-serif"
+      }
+    },
+    "font-size": {
+      "base": {
+        "value": "16px",
+        "comment": "Base font size for design system in px"
+      },
+      "40": {
+        "value": "2.5rem"
+      },
+      "32": {
+        "value": "2rem"
+      },
+      "24": {
+        "value": "1.5rem"
+      },
+      "18": {
+        "value": "1.125rem"
+      },
+      "16": {
+        "value": "1rem"
+      },
+      "14": {
+        "value": "0.875rem"
+      },
+      "12": {
+        "value": "0.75rem"
+      },
+      "11": {
+        "value": "0.688rem"
+      }
+    },
+    "font-weight": {
+      "light": {
+        "value": "400"
+      },
+      "medium": {
+        "value": "400",
+        "comment": "Arimo weight, should be updated to 500"
+      },
+      "bold": {
+        "value": "700",
+        "comment": "Arimo weight, should be updated to 600"
+      }
```

Comment:
The new ramp is designed for Graphik, which is more visually distinct between different font weights. This preserves existing appearances